sdk/test-utils/perfstress/src/utils.ts

```diff
@@ -14,3 +14,17 @@ export function getEnvVar(name: string) {
   }
   return val;
 }
+
+/**
+ * Reads a readable stream. Doesn't save to a buffer.
+ *
+ * @export
+ * @param {NodeJS.ReadableStream} stream A Node.js Readable stream
+ */
+export async function readStream(stream: NodeJS.ReadableStream) {
```

Comment:
Could we rename this to something like `drainStream` so it's more clear that we're just throwing away the bytes?